airlock/middleware.py

```diff
@@ -32,6 +33,8 @@ def __call__(self, request):
                     user = User.from_session(request.session)
 
         request.user = user
+        span = trace.get_current_span()
+        span.set_attribute("user", user.username if user else "")
```

Comment:
Worth using `conftests.get_trace()` and testing this is set correctly in tests/integration/test_middleware.py?